### PR TITLE
Feature/auto setup intercom

### PR DESCRIPTION
**Pull request by unity**
Todo: 
- [ ] Merge to Master
- [ ] Check everyone is using `hull-intercom.herokuapp.com`
- [ ] Refresh manifest for all installed instances

### CODE REVIEW

File: src/ship.js [hunk 1]
```diff
@@ -15,8 +15,17 @@
...
+    window.Intercom && window.Intercom("trackEvent", event, params)
+  }
+  function identify(payload){
+    window.Intercom && window.Intercom("update", payload)
+  }
+
   if (window.Hull) {
     Hull.onEmbed(function onEmbed(rootNode, deployment, hull) {
+      Hull.on('hull.track', track);
+      Hull.on('hull.identify', identify);
```
Comment (timliuhull) [inline src/ship.js]: Have you tested this?  No way to add tests for it right?

Comment (timliuhull) [reply src/ship.js]: There's ship.js files in other connectors too right? Are they supposed to all be the same?

**Comment by unity** [comment]
No way to add tests. This is pretty safe since we are adding event listeners to Hull.js which is the very library that installed the intercom connector in the page in the first place. So we can rely on it to be available and installed already.

**Comment by unity** [comment]
Ship.js files are client-side components of the connector that Hull.js can bring in the page. This is done by going to the Platform screen in the dashboard and adding the connector to the platform.

**Pull request merged**